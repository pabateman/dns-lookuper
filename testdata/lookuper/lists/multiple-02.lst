spotify.com
cncf.io






another/invalid/name
