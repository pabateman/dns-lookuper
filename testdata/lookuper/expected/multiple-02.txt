23.185.0.3
35.186.224.24
